lib/erl_docgen/priv/bin/validate_links.escript

```diff
@@ -3,7 +3,7 @@
 %%! +A 1 +SDio 1 +S 1 -mode minimal
 %% %CopyrightBegin%
 %%
-%% Copyright Ericsson AB 2020-2023. All Rights Reserved.
+%% Copyright Ericsson AB 2020-2021. All Rights Reserved.
```

Comment:
Why is this backdated to 2021? :thinking:

Reply:
I originally did the changes in an older version and then copied the file over and then the old copyright was accidentally added to the change. Will remove it, thanks for noticing :)